hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AFileSystemIsolatedClassloader.java

```diff
@@ -77,19 +109,9 @@ private void assertInNewFilesystem(Map<String, String> confToSet, Consumer<FileS
     }
   }
 
-  private Map<String, String> mapOf() {
-    return new HashMap<>();
-  }
-
-  private Map<String, String> mapOf(String key, String value) {
-    HashMap<String, String> m = new HashMap<>();
-    m.put(key, value);
-    return m;
-  }
```

Comment:
Since I added test cases that set 2 key-value pairs, I switched to `Map.of` instead of extending these. Not sure if there was a reason to avoid `Map.of` here.

Reply:
Thanks, makes sense!